packages/rich-text/package.json

```diff
@@ -27,29 +27,29 @@
     "prepublishOnly": "yarn build"
   },
   "dependencies": {
-    "@contentful/contentful-slatejs-adapter": "^14.2.0",
+    "@contentful/contentful-slatejs-adapter": "^15.3.5",
     "@contentful/field-editor-reference": "^2.20.6",
-    "@contentful/forma-36-react-components": "^3.97.0",
-    "@contentful/rich-text-types": "^15.3.3",
-    "@udecode/plate-break": "^3.0.3",
-    "@udecode/plate-common": "^2.0.0",
-    "@udecode/plate-core": "^1.0.0",
-    "@udecode/plate-html-serializer": "^2.0.0",
-    "@udecode/plate-list": "^3.0.4",
-    "@udecode/plate-paragraph": "^2.0.0",
-    "@udecode/plate-table": "^3.1.2",
-    "@udecode/plate-trailing-block": "^2.0.0",
+    "@contentful/forma-36-react-components": "^3.98.3",
+    "@contentful/rich-text-types": "^15.3.6",
+    "@udecode/plate-break": "^3.2.0",
+    "@udecode/plate-common": "^3.2.0",
+    "@udecode/plate-core": "^3.2.0",
+    "@udecode/plate-html-serializer": "^3.2.0",
+    "@udecode/plate-list": "^3.2.0",
+    "@udecode/plate-paragraph": "^3.2.0",
+    "@udecode/plate-table": "^3.2.0",
+    "@udecode/plate-trailing-block": "^3.2.0",
     "fast-deep-equal": "^3.1.3",
     "react": ">=16.14.0",
-    "slate": "^0.65.3",
-    "slate-history": "^0.65.3",
-    "slate-hyperscript": "^0.62.0",
+    "slate": "^0.66.1",
+    "slate-history": "^0.66.0",
+    "slate-hyperscript": "^0.66.0",
     "slate-react": "^0.65.3"
```

Comment:
Upgrading to `slate-react > v0.65.3` [breaks the undo/redo functionality](https://github.com/ianstormtaylor/slate/issues/4530)